### PR TITLE
Check sep24 interactive URL

### PR DESCRIPTION
**Pull request by gianfra-t**
Related to #128.

We realize this will not fix the issue the user may see from waiting too much to click the second confirmation button.

This changes will refresh the sep24 url every 20 seconds until the user actually clicks on the button.

### CODE REVIEW

File: src/hooks/useMainProcess.ts [hunk 1]
```diff
@@ -107,36 +110,33 @@ export const useMainProcess = () => {
...
+          };
 
-          updateHookStateFromState(initialState);
+          const executeFinishInitialState = async () => {
+            try {
+              // This will only return if the initialization of the offramping process is finished or exited because an error occurred
+              await fetchAndUpdateSep24Url();
+            } catch (error) {
+              console.error('Some error occurred finalizing the initial state of the offramping process', error);
+              setOfframpingStarted(false);
```
[the remark this answers is not in the document]
Comment (gianfra-t) [reply src/hooks/useMainProcess.ts:146]: No, you are right for the case there was an error on `fetchAndUpdateSep24Url`. That's why I refactored a bit by adding this [clean function](https://github.com/pendulum-chain/vortex/pull/155/files#diff-fedb8a30074ce22b1fb6aa02c40daa53038ea2a73c885a7bd7db6f5dfe5e8efaR84) that will take care of resetting the states on both scenarios: Failure on sep24First and when the user opens the anchor kyc. 

Or do you think that even with this addition the interval will not be properly cleared? I tested it and mocking an error seems to clear the interval and variables properly.

[the remark this answers is not in the document]
Comment (TorstenStueber) [reply src/hooks/useMainProcess.ts:146]: Ah, I missed the clean function. Then it should work. 👍

**Review by TorstenStueber — APPROVED** [review]
Now it should be working correctly. Thanks.

Just something important: I had to [merge a fix in the mean time](https://github.com/pendulum-chain/vortex/pull/180), can you check that this get cleanly incorporated in your PR so that after merging the fix would not regress?

**Comment by gianfra-t** [comment]
@TorstenStueber it merged well. I imagine that the behavior of the label "Confirming" now that we refresh the `sep24` first process is that upon first successful response, we keep display of "Enter details".

**Review by TorstenStueber — APPROVED** [review]

**Comment by TorstenStueber** [comment]
I tested, works correctly.

**Pull request merged**